render-utils.js

```diff
@@ -1,13 +1,13 @@
-export function renderTodo(todo) {
-    // create a div and a p tag
-    
-    // depending on whether the todo is complete, give the div the appropriate css class ('complete' or 'incomplete')
-
-    // add the 'todo' css class no matter what
+// create a div and a p tag
+// depending on whether the todo is complete, give the div the appropriate css class ('complete' or 'incomplete')
```

Comment:
The div will need a classList added to it. If complete, add css class 'complete', if incomplete add 'incomplete'